src/svcompleter.ts

```diff
@@ -29,133 +30,558 @@ import {
+    ["endspecify"],
+    ["endsequence"],
+    ["endtable"],
+    ["endtask"],
+    ["enum"],
+    ["event"],
+    ["expect"],
+    ["export"],
+    ["extends"],
+    ["extern"],
+    ["final"],
+    ["first_match"],
+    ["for"],
+    ["force"],
+    ["foreach"],
+    ["forever"],
+    ["fork"],
+    ["forkjoin"],
+    ["function"],
+    ["generate"],
+    ["genvar"],
+    ["highz0"],
+    ["highz1"],
+    ["if"],
+    ["iff"],
+    ["ifnone"],
+    ["ignore_bins"],
+    ["illegal_bins"],
+    ["import"],
+    ["incdir"],
+    ["include"],
+    ["initial"],
+    ["inout"],
+    ["input"],
+    ["inside"],
+    ["instance"],
+    ["int"],
+    ["integer"],
+    ["interface"],
+    ["intersect"],
+    ["join"],
+    ["join_any"],
+    ["join_none"],
+    ["large"],
+    ["liblist"],
+    ["library"],
+    ["local"],
+    ["localparam"],
+    ["logic"],
+    ["longint"],
+    ["macromodule"],
+    ["matches"],
+    ["medium"],
+    ["modport"],
+    ["module"],
+    ["nand"],
+    ["negedge"],
+    ["new"],
+    ["nmos"],
+    ["nor"],
+    ["noshowcancelled"],
+    ["not"],
+    ["notif0"],
+    ["notif1"],
+    ["null"],
+    ["or"],
+    ["output"],
     ["package"],
-    ["endpackage"],
+    ["packed"],
+    ["parameter"],
+    ["pmos"],
+    ["posedge"],
+    ["primitive"],
+    ["priority"],
+    ["program"],
+    ["property"],
+    ["protected"],
+    ["pull0"],
+    ["pull1"],
+    ["pulldown"],
+    ["pullup"],
+    ["pulsestyle_onevent"],
+    ["pulsestyle_ondetect"],
+    ["pure"],
+    ["rand"],
+    ["randc"],
+    ["randcase"],
+    ["randsequence"],
+    ["rcmos"],
+    ["real"],
+    ["realtime"],
+    ["ref"],
+    ["reg"],
+    ["release"],
+    ["repeat"],
+    ["return"],
+    ["rnmos"],
+    ["rpmos"],
+    ["rtran"],
+    ["rtranif0"],
+    ["rtranif1"],
+    ["scalared"],
+    ["sequence"],
+    ["shortint"],
+    ["shortreal"],
+    ["showcancelled"],
+    ["signed"],
+    ["small"],
+    ["solve"],
+    ["specify"],
+    ["specparam"],
+    ["static"],
+    ["string"],
+    ["strong0"],
+    ["strong1"],
+    ["struct"],
+    ["super"],
+    ["supply0"],
+    ["supply1"],
+    ["table"],
+    ["tagged"],
+    ["task"],
+    ["this"],
+    ["throughout"],
+    ["time"],
+    ["timeprecision"],
+    ["timeunit"],
+    ["tran"],
+    ["tranif0"],
+    ["tranif1"],
+    ["tri"],
+    ["tri0"],
+    ["tri1"],
+    ["triand"],
+    ["trior"],
+    ["trireg"],
+    ["type"],
+    ["typedef"],
+    ["union"],
+    ["unique"],
+    ["unsigned"],
+    ["use"],
+    ["uwire"],
+    ["var"],
+    ["vectored"],
+    ["virtual"],
+    ["void"],
+    ["wait"],
+    ["wait_order"],
+    ["wand"],
+    ["weak0"],
+    ["weak1"],
+    ["while"],
+    ["wildcard"],
+    ["wire"],
+    ["with"],
+    ["within"],
+    ["wor"],
+    ["xnor"],
+    ["xor"]
 ];
+let sv_completion_keyword_items = [];
+for (let n = 0; n < sv_completion_keywords.length; n++) {
+    sv_completion_keyword_items.push({
+        label: sv_completion_tick[n][0],
```

Comment:
Typo. Needs to be sv_completion_keywords

Reply:
Fixed.